test/e2e/network/kube_proxy.go

```diff
@@ -97,43 +95,17 @@ var _ = SIGDescribe("Network", func() {
 				Containers: []v1.Container{
 					{
 						Name:            "e2e-net-exec",
-						Image:           kubeProxyE2eImage,
+						Image:           imageutils.GetE2EImage(imageutils.DebianIptables),
 						ImagePullPolicy: v1.PullIfNotPresent,
-						Args:            []string{"pause"},
-						VolumeMounts: []v1.VolumeMount{
-							{
-								Name:      "proc-net",
-								MountPath: "/rootfs/proc/net",
-								ReadOnly:  true,
-							},
-						},
+						Command:         []string{"sleep", "600"},
 						SecurityContext: &v1.SecurityContext{
 							Privileged: &privileged,
 						},
 					},
 				},
-				Volumes: []v1.Volume{
-					{
-						Name: "proc-net",
-						VolumeSource: v1.VolumeSource{
-							HostPath: &v1.HostPathVolumeSource{
-								Path: "/proc/net",
-							},
-						},
-					},
-				},
 			},
 		}
 		fr.PodClient().CreateSync(hostExecPod)
-		defer fr.PodClient().DeleteSync(hostExecPod.Name, metav1.DeleteOptions{}, framework.DefaultPodDeletionTimeout)
-
-		// Some distributions (Ubuntu 16.04 etc.) don't support the proc file.
```

Comment:
we get rid of this check because the `debian-iptables` image has the conntrack tool.
Conntrack is a dependency of kubernetes, so it has enabled in the kernel